File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/ui/PassedCoursesFragment.java
package com.vladislavmyasnikov.courseproject.ui;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.vladislavmyasnikov.courseproject.R;

public class PassedCoursesFragment extends Fragment {

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_passed_courses, container, false);
    }

    public static PassedCoursesFragment newInstance() {
        return new PassedCoursesFragment();
    }
}

File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/ui/courses/PassedCoursesFragment.kt
package com.vladislavmyasnikov.courseproject.ui.courses

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.vladislavmyasnikov.courseproject.R

class PassedCoursesFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        return inflater.inflate(R.layout.fragment_passed_courses, container, false)
    }



    companion object {

        fun newInstance(): PassedCoursesFragment {
            return PassedCoursesFragment()
        }
    }
}
